src/ui/js/Env.js

```diff
@@ -498,6 +498,16 @@ Env.buildProfileLink = function(playerName, textOnly) {
   }
 };
 
+// Utility function to build a vacation image object
+Env.buildVacationImage = function(size) {
+  var image = (size=='large') ? 'vacation22.png' : 'vacation16.png';
+  return  $('<img>', {
+    'src': Env.ui_root + 'images/'+image,
```

Comment:
Nit (don't bother to fix unless i find actual problems): i feel strongly about horizontal whitespace around code symbols, as around the second + here.